pkg/model/dev.go

```diff
@@ -635,6 +639,17 @@ func (dev *Dev) setTimeout() error {
 	return nil
 }
 
+func (dev *Dev) validateSync() error {
```

Comment:
There should be some tests added for this function. It would cover the case mentioned by Javi.

Reply:
Sure. Tests will be added after I fix an underlying issue. 

Javier & I are on that!

Reply:
This function is now outdated. I have added the validation to the `validat()` itself and modified the tests.